sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -6,68 +6,99 @@
           <slot name="coeSummarySubtitle">{{ coeSummarySubtitle }}</slot>
         </template>
         <template #actions>
-          <v-text-field
-            density="compact"
-            label="Search Name"
-            variant="outlined"
-            v-model="searchCriteria"
-            data-cy="searchCriteria"
-            @keyup.enter="searchCOE"
-            prepend-inner-icon="mdi-magnify"
-            hide-details="auto"
-          />
+          <v-row class="justify-end">
+            <v-col cols="auto">
+              <v-btn-toggle
+                v-model="intensityFilter"
+                class="btn-toggle"
+                selected-class="selected-btn-toggle"
+                @update:model-value="resetPageAndLoadEnrollments"
+              >
+                <v-btn
+                  rounded="xl"
+                  color="primary"
+                  :value="IntensityFilter.All"
+                  class="mr-2"
+                  >All</v-btn
+                >
+                <v-btn
+                  v-for="intensity in Object.values(OfferingIntensity)"
+                  :key="intensity"
+                  rounded="xl"
+                  color="primary"
+                  :value="intensity"
+                  class="mr-2"
+                  >{{ mapOfferingIntensity(intensity) }}</v-btn
+                >
+              </v-btn-toggle>
+            </v-col>
+            <v-col cols="3">
```

Comment:
Can we remove the `cols="3"` here to allow the search to take the entire space, as shown below?

<img width="1621" height="597" alt="image" src="https://github.com/user-attachments/assets/769b69b4-c02c-4cf1-8451-c215bc4af197" />